models/engine/file_storage.py

```diff
@@ -68,3 +68,19 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Retrieve one object"""
+        if (cls is None) or (str(cls) not in classes) or id is None:
```

Comment:
`id is None` is not between round brackets, while the other conditions are >:[

Reply:
I will take in consideration your comments, but to be honest, i can't promise anything ;0